config/routes.rb

```diff
@@ -8,11 +8,22 @@
   # get '/user/:id', to: 'user#show', as: 'user'
   # get '/post/:user_id/posts', to: 'post#index', as: 'user_posts'
   # get '/post/:user_id/posts/:id', to: 'post#show', as: 'user_post'
+
   root 'users#index'
   resources :users, only: %i[show] do
     resources :posts, only: %i[index show new create destroy] do
       resources :comment, only: %i[new index create update destroy]
       resources :like, only: %i[create]
     end
   end
+  # end
+
+  namespace :api do
+    namespace :v1 do
+      post 'users/sign_in' => 'users#login'
```

Comment:
- [ ] Based on the requirements of this project we should have an API endpoint for registration. I couldn't be registered and hence could not also sign in.  
- With the issue above, I couldn't create a post, get all posts, add(POST) comments to a post, or get all comments. Kindly make are that all the endpoints return results in JSON format.